### PR TITLE
Fix to error forward with inline image

### PR DESCRIPTION
**Pull request by StCyr**
fixes https://github.com/nextcloud/mail/issues/4656

### CODE REVIEW

File: src/components/Composer.vue [hunk 1]
```diff
@@ -463,7 +463,7 @@ export default {
 
 		// Add attachments in case of forward
```
Comment (ChristophWurst) [inline src/components/Composer.vue:464]: please add a note about why we exclude inline attachments here

File: src/components/MessageAttachments.vue [hunk 2]
```diff
@@ -22,17 +22,18 @@
...
-				:id="attachment.id"
-				:key="attachment.id"
-				:file-name="attachment.fileName"
-				:size="attachment.size"
-				:url="attachment.downloadUrl"
-				:is-image="attachment.isImage"
-				:is-calendar-event="attachment.isCalendarEvent"
-				:mime="attachment.mime"
-				:mime-url="attachment.mimeUrl" />
+			<div v-for="attachment in attachments" :key="attachment.id">
```
Comment (ChristophWurst) [inline src/components/MessageAttachments.vue:25]: is this really necessary?

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: src/components/Composer.vue (hunk 1 above)
Comment (StCyr) [reply src/components/Composer.vue:464]: done

File: src/components/MessageAttachments.vue (hunk 2 above)
Comment (StCyr) [reply src/components/MessageAttachments.vue:25]: No indeed.

Comment (StCyr) [reply src/components/MessageAttachments.vue]: changed

**Review by ChristophWurst — APPROVED** [review]
Please squash :)

**Pull request merged**